README.md

```diff
@@ -150,20 +150,49 @@ We collaborate closely with the design and product team. The design team provide
 
 ## Deployment
 
+### Git Branching Model
+
+- When a Github issue is started, create a branch with the `feature/` prefix. Every push to branches with this prefix will automatically deploy to the review AWS environment. See the [Review branches](#review-branches) section for details on how to access this build.
+
+- After a pull request for a `feature/` branch is approved, it is merged into the `staging` branch. This will trigger an automatic deployment to the staging AWS environment. Note that changes may take several minutes to take effect. The staging app is accessible at http://staging.fightpandemics.work.
+
+- When we are ready to release to production, a pull request is opened to merge the `staging` branch to `production`. After this pull request is approved and merged, it will trigger an automatic deployment to the production AWS environment. Note that changes may take several minutes to take effect. The production app is accessible at http://fightpandemics.com
```

Comment:
should we change the prod urls to `https://fightpandemics.com`? This will work too though

Reply:
Either works, but probably best practice to use HTTPS